### PR TITLE
Feature - Refresh for home

### PR DESCRIPTION
**Pull request by at-hanguyen3**
- [x] Refresh for home

### CODE REVIEW

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controller/Home/HomeViewController.swift [hunk 1]
```diff
@@ -275,5 +290,6 @@ extension HomeViewController {
         static let heightLocationView: CGFloat = UIApplication.shared.statusBarFrame.height + 40
         static let buttonNormalFont = UIFont.systemFont(ofSize: 14)
         static let buttonSelectedFont = UIFont.boldSystemFont(ofSize: 15)
+        static let refresh: String = "Pull to refresh"
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/Home/HomeViewController.swift]: NOt need

File: MyApp/View/Controller/Home/HomeViewController.swift [hunk 2]
```diff
@@ -128,6 +130,19 @@ final class HomeViewController: ViewController {
         collectionView.setContentOffset(.zero, animated: false)
     }
 
+    func configRefreshControl() {
+        refreshControl = UIRefreshControl()
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/Home/HomeViewController.swift]: Not need

File: MyApp/View/Controller/Home/HomeViewController.swift [hunk 3]
```diff
@@ -30,6 +30,7 @@ final class HomeViewController: ViewController {
 
     // MARK: - Properties
     private var viewModel = HomeViewModel()
+    var refreshControl = UIRefreshControl()
```
Comment (at-hainguyen3) [inline MyApp/View/Controller/Home/HomeViewController.swift]: Private

**Pull request merged**